### ISSUE TITLE
PermissionError AppData\crtFile, but including DLLs didn't seem to help

### ISSUE THREAD

**Issue by AaronAikman** [pyinstaller/pyinstaller#3745]
I had been using PyInstaller 3.x with Python 3.6 on Windows 7 with great success for a number of weeks.  I recently made a change and sought to rebuild the exe.  It failed citing some crt variant of 
`PermissionError: [Errno 13] Permission denied: 'C:\\Users\\Hermes\\AppData\\Roaming\\pyinstaller\\bincache00_py36_64bit\\api-ms-win-crt-runtime-l1-1-0.dll'`

At first I tried different python versions.  
I tried building the previous python file, which did not work. 
 Then I made sure I had the upgraded PyInstaller to 3.4. 
 I tried running my bat file in admin mode. 
 I tried clearing out the cache folder. 
 I tried to make sure the write permissions were correct for the cache files. 
 I tried using my last successful build folder.  
I tried adding the crt dlls as found in my system folder.  
I tried a number of argument combinations. 
Based upon [https://pyinstaller.readthedocs.io/en/v3.4/usage.html#windows](https://pyinstaller.readthedocs.io/en/v3.4/usage.html#windows),
 I tried add-binary with C:\Program Files (x86)\Windows Kits\10\Redist\ucrt\DLLs. 
 I tried copying that DLLs folder to folder I was running out of, which is the most recent argument I used
`--add-binary DLLs;.`
I tried pip 'install https://github.com/pyinstaller/pyinstaller/archive/develop.zip'
I tried with a minimal python file

Here is the link to my output

[https://gist.github.com/AaronAikman/27acb05944871f5e69d0316ac6b60812](https://gist.github.com/AaronAikman/27acb05944871f5e69d0316ac6b60812)

Here is the original command I had been using with success
`C:\Python36\python.exe -m PyInstaller --clean --win-private-assemblies --onefile --noconsole --icon=jsonitor.ico JSONitor.py`

And here is the last command I tried when attempting to include necessary dlls for Windows < 10
`C:\Python36\python.exe -m PyInstaller --runtime-tmpdir=F:\Database\Satchel\Works\Scripting\JSONitorBuild\Solo\tmp --add-binary DLLs;. --clean --win-private-assemblies --onefile --debug=all --log-level=DEBUG > out.txt --icon=F:\Database\Satchel\Works\Scripting\JSONitorBuild\Solo\jsonitor.ico F:\Database\Satchel\Works\Scripting\JSONitorBuild\Solo\JSONitor.py`

Here is my most recent spec file

`# -*- mode: python -*-

block_cipher = None

a = Analysis(['F:\\Database\\Satchel\\Works\\Scripting\\JSONitorBuild\\Solo\\JSONitor.py'],
             pathex=['f:\\Database\\Satchel\\Works\\Scripting\\JSONitorBuild\\Solo'],
             binaries=[('DLLs', '.')],
             datas=[],
             hiddenimports=[],
             hookspath=[],
             runtime_hooks=[],
             excludes=[],
             win_no_prefer_redirects=False,
             win_private_assemblies=True,
             cipher=block_cipher,
             noarchive=False)
pyz = PYZ(a.pure, a.zipped_data,
             cipher=block_cipher)
exe = EXE(pyz,
          a.scripts,
          a.binaries,
          a.zipfiles,
          a.datas,
          [],
          name='JSONitor',
          debug=False,
          bootloader_ignore_signals=False,
          strip=False,
          upx=True,
          runtime_tmpdir='F:\\Database\\Satchel\\Works\\Scripting\\JSONitorBuild\\Solo\\tmp',
          console=True , icon='F:\\Database\\Satchel\\Works\\Scripting\\JSONitorBuild\\Solo\\jsonitor.ico')`

Any help with this would be greatly appreciated.  Thank you for your time.

**Comment by AaronAikman** [comment]
Is there any additional information I could provide that would help with this issue?

**Comment by htgoebel** [comment]
Well, this is is "permission error", which is outside of PyInstallers capabilities. You need to check the permissions again. maybe the traceback will give you more insight, what caused the error.

BTW: Most of what you tied has no, absolutely no effect on an permission error. Why should using a bat-file, changing order of options, or a different Python version have any effect on a permission error in one of your personal files?

**Comment by AaronAikman** [comment]
I did not try changing the order of options.  I tried excluding certain arguments or including others.  
I just thought I'd provide information about the approaches I tried in hopes that pyinstaller wouldn't encounter that error.  I would have expected --runtime-tmpdir to "ignore any temp-folder location defined by the run-time OS", but I didn't see that happening.  Did I use that argument incorrectly?
Thanks for responding.
